docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -310,18 +310,29 @@ The POST request body should be in JSON format with the following structure:
 
 Used to lookup fungible assets balances
 
-`GET /v1/account/{address}/balance`
+`POST /v1/account/{address}/balance`
 
 #### Path parameters
 
 * `address` - The address for lookup. eg. `0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
 
-#### Query parameters
+#### Request body:
+
+The POST request body should be in JSON format with the following structure:
+
+```typescript
+{
+    "projectId": string,
+    "currency": string,
+    "chain": (optional) string,
+    "forceUpdate": (optional) string[]
```

Comment:
Use actual TypeScript here, is it

```suggestion
    "chain"?: string,
    "forceUpdate"?: string[]
```

or

```suggestion
    "chain": string | null,
    "forceUpdate": string[] | null
```

Reply:
Thanks for pointing out this! It's changed.